doc/content/verification/ver-1a.md

```diff
@@ -2,47 +2,148 @@
+
+!media figures/ver-1a_schematic.png
+    style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+    id=ver-1a_schematic
+    caption= Schematic of the verification 1a case and illustration of the quantities used for verification.
+
+!table id=ver-1a_set_up_values caption=Values of material properties and case geometry with $R$, the gas constant, as defined in [PhysicalConstants](source/utils/PhysicalConstants.md).
+| Parameter | Description                | Value                                  | Units      |
+| --------- | -------------------------- | -------------------------------------- | ---------- |
+| $V$       | Enclosure volume           | 5.20e-11                               | m$^3$      |
+| $A$       | SiC slab surface area      | 2.16e-6                                | m$^2$      |
+| $P_0$     | Initial enclosure pressure | 1e6                                    | mPa        |
+| $T$       | Temperature                | 2373                                   | K          |
+| $D$       | Tritium diffusivity in SiC | 1.58e-4 $\times \exp(-308000.0/(R*T))$ | m$^2$/2    |
+| $S$       | Tritium solubility in SiC  | 7.244e22 / $T$                         | 1/m$^3$/Pa |
+| $l$       | Slab thickness             | 3.30e-5                                | m          |
+
+## Verification of the release fraction on the outer surface of the SiC slab (TMAP4)
+
+### Analytical solution
+
+In [!cite](longhurst1992verification), i.e., TMAP4, the verification test focused on the fractional release from the outside of the slab. The analytical solution for this quantity is given by
 
 \begin{equation}
-    FR = 1.0 - \sum_{n=1}^{\infty} \frac{2\ L sec \ \alpha_{n} - \exp\left(\frac{-\alpha_{n}^2 D T}{l^{2}}\right)}{L(L+1) + \alpha_n^{2}}
+    FR(t) = 1.0 - \sum_{n=1}^{\infty} \frac{2\ L \sec (\alpha_{n}) \exp\left(-\alpha_{n}^2[D t/l^{2}]\right)}{L(L+1) + \alpha_n^{2}},
 \end{equation}
 
 where
 
 \begin{equation}
     L = \frac{lA}{V \phi}
 \end{equation}
+with
+\begin{equation}
+    \phi = \frac{\text{source concentration}}{\text{layer concentration}} = \frac{1}{S k_b T},
+\end{equation}
+
+where the "source concentration" is the concentration in the enclosure ($P(t)/k_b/T$) with $k_b$ is the Boltzmann constant (as defined in [PhysicalConstants](source/utils/PhysicalConstants.md)), and "layer concentration" is the concentration in the slab at the interface with the enclosure ($S P(t)$). $\phi$ is constant in time. $\alpha_n$ are the roots of
```

Comment:
```suggestion
where the "source concentration" is the concentration in the enclosure ($P(t)/k_b/T$), $k_b$ is the Boltzmann constant (as defined in [PhysicalConstants](source/utils/PhysicalConstants.md)), and "layer concentration" is the concentration in the slab at the interface with the enclosure ($S P(t)$). $\phi$ is constant in time. $\alpha_n$ are the roots of
```

Reply:
I think both work since `$P(t)/k_b/T$` uses `k_b` as well, but I'll commit this.